subsys/bluetooth/services/wifi_prov/proto/version.proto

```diff
@@ -9,5 +9,5 @@ option java_multiple_files = true;
 option java_package = "no.nordicsemi.android.wifi.provisioning";
 
 message Info {
-  required uint32 version         = 1;
+  required uint32 version         = 2;
```

Comment:
why the extra spacing ?
```suggestion
  required uint32 version = 2;
```

Reply:
It's an existing change, left it to avoid making whitespace changes.